tests/unit/dataactbroker/test_dashboard_handler.py

```diff
@@ -240,28 +240,28 @@ def setup_submissions(sess, admin=False):
                        pub_month_sub_a1])
 
     # Setup certified error metadata
```

Comment:
certified -> published

Also suggest updating the variables (`cert_` -> `pub_`). The last one can be just `pub_month_sub_a1`.